responsibleai/responsibleai/_managers/causal_manager.py

```diff
@@ -3,13 +3,13 @@
 
 """Manager for causal analysis."""
 import numpy as np
-import pandas as pd
 
 from typing import Any, List, Optional, Union
 
 from econml.solutions.causal_analysis import CausalAnalysis
 from pathlib import Path
 
+from responsibleai._data_validations import _validate_train_test_categories
```

Comment:
If we're going to use `_validate_train_test_categories` outside of `_data_validations.py`, then we should make it `validate_train_test_categories` with no underscore

Reply:
Renamed it.